base/libgit2/tree.jl

```diff
@@ -16,19 +15,22 @@ function treewalk(f::Function, tree::GitTree, payload=Any[], post::Bool = false)
     return cbf_payload
 end
 
-"Get the filename of a tree entry."
+"""Returns a file name, as a `String`, of a tree entry.
+
+In case of error, `nothing` is returned.
```

Comment:
returning the empty string is also not that satisfying, what might cause this to error?